pkgs/applications/editors/vscode/extensions/vadimcn.vscode-lldb/default.nix

```diff
@@ -38,6 +38,13 @@ let
       "--bin=codelldb"
     ];
 
+    postFixup = ''
+      wrapProgram $out/bin/codelldb \
+        --set-default LLDB_DEBUGSERVER_PATH "${lldb.out}/bin/lldb-server"
+      ln -s ${lldb.lib} $out/lldb
+      ln -s $out/lib/libcodelldb.so $out/bin/libcodelldb.so
```

Comment:
libs should not be under bin

Reply:
That appears to work.
@Majiir does this work for you in VSCode?

Reply:
The plugin fails to build. I think you need to copy only `${adapter}/share/*` instead of `${adapter}/{share,lib}/*`, since there are files with the same name in both.

Reply:
Thanks - Should be fixed now.

Reply:
Builds and works in VSCode. :rocket:

Reply:
Thanks for testing!